packages/wrangler/src/metrics/types.ts

```diff
@@ -0,0 +1,114 @@
+export type CommonEventProperties = {
+	/** The version of the Wrangler client that is sending the event. */
+	wranglerVersion: string;
+	/**
+	 * The platform that the Wrangler client is running on.
+	 */
+	osPlatform: string;
+	/**
+	 * The platform version that the Wrangler client is running on.
+	 */
+	osVersion: string;
+	/**
+	 * The package manager that the Wrangler client is using.
+	 */
+	packageManager: string | undefined;
+	/**
+	 * The major version of node that the Wrangler client is running on.
+	 */
+	nodeVersion: number;
+	/**
+	 * Whether this is the first time the user has used the wrangler client.
+	 */
+	isFirstUsage: boolean;
+	/**
+	 * What format is the configuration file? No content from the actual configuration file is sent.
+	 */
+	configFileType: "toml" | "json" | "jsonc" | "none" | "invalid";
```

Comment:
The `configFormat` helper we migrated will only return 3 different types

```suggestion
	configFileType: "toml" | "jsonc" | "none";
```

Reply:
1988a21d955335565bfd281c0e5f184c8d381a03